tests/integration/api_packages_npm_test.go

```diff
@@ -35,6 +35,8 @@ func TestPackageNpm(t *testing.T) {
 	packageTag2 := "release"
 	packageAuthor := "KN4CK3R"
 	packageDescription := "Test Description"
+	packageBinName := "cli"
+	packageBinPath := "cli.sh"
```

Comment:
```suggestion
	packageBinPath := "./cli.sh"
```

I think this format is more commonplace, but both should work.